examples/README.md

```diff
@@ -0,0 +1,54 @@
+# examples
+
+This directory contains example code for both installing Contour Envoy.
+ also for setting up various extensions and other pieces of software that may help with running Contour.
+
+Most of the directories contain a complete set of Kubernetes YAML that can be applied to a cluster. This section describes the purpose of each directory, and inside each directory is a README.md that describes the details.
+
+## [`contour`](./contour/README.md)
+
+This is the recommended example installation of Contour.
+It will deploy Contour into a Deployment, and Envoy into a Daemonset.
+The gRPC communication is secured with certificates.
+A `LoadBalancer` Service is creatd to expose Envoy to your upstream load balancer provider.
+
+## [`contour-dev`](./contour-dev/README.md)
+
+This is an example installation of Contour, designed for local development or other trial use.
```

Comment:
For local development. period

lets just document what we want people to use it for.